src/core/core.tooltip.js

```diff
@@ -101,7 +101,7 @@ module.exports = function(Chart) {
 
 	// Helper to push or concat based on if the 2nd parameter is an array or not
 	function pushOrConcat(base, toPush) {
-		if (toPush) {
+		if (toPush !== undefined && toPush !== null) {
```

Comment:
@JemarJones sorry for the delay, can you rebase your PR and use the new `if(!helpers.isNullOrUndef(toPush))`?
@etimberg are you ok with that change and accept numbers for the label callback?